keanu-python/keanu/plots/autocorrelation.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from keanu import stats
+from typing import Tuple, Any
+
+
+def _import_matplotlib() -> Any:
+    try:
+        import matplotlib.pyplot as plt
+    except:
+        raise ImportError("Could not find Matplotlib")
+    return plt
+
+
+def _create_new_mpl() -> Tuple[Any, Any]:
```

Comment:
You can use double underscore for all these methods, since they are not called in any other module.